department-of-veterans-affairs/abd-vro
# Feature - add lookup table + unit test

## What was the problem?

- Previously the API always returned the code for asthma. Now it should return the classification mapped to the diagnostic code provided (if applicable)

Associated tickets or Slack threads:

- [Integrate lookup table in Python endpoint #160](https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/160)
- [Create unit tests for API endpoint #161](https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/161)

## How does this fix it?[^1]

- adds lookup table and BRD data as .json files
- adds python code for creating a dictionary based on the lookup table json
- updates existing python endpoint to pull from the new lookup table

## How to test this PR
- See [Unit Tests in README.md](https://github.com/department-of-veterans-affairs/abd-vro/tree/feature-unit-tests/domain-cc#unit-tests)


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
